app/main/routes.py

```diff
@@ -127,6 +127,30 @@ def poc_search():
     )
 
 
+@bp.route("/poc-browse-series", methods=["POST", "GET"])
+@access_token_sign_in_required
+def poc_browse_series():
+    form = SearchForm()
+    search_results = []
+    series = request.form.get("series", "").lower()
```

Comment:
If there is a separate route for series browse then we dont need the "series" query param - would only need it if we share one browse view.

@colinbowen has @Terry-Price spoken to you about how the consignmen vs series browse pages will differ in that sense?

Reply:
poc_browse_series will be redundant and super seeded by front end browse route designed by Colin